Libraries/NewAppScreen/index.js

```diff
@@ -11,53 +11,18 @@
 'use strict';
```

Comment:
This file is now a duplicate of `template/App.js`. Is it worth maintaining both files? Their imports are different, given where each module lives, but it will create extra overhead in maintaining these.

Reply:
No we should kill it and replace it with a module that just exports everything that is needed by App.js. The template should have only a single import that destructures into everything that's needed there.

Reply:
Should the usage of this in RNTester then become a rendering of each component from this module?

https://github.com/facebook/react-native/blob/2cd6c0c82e0d01201d97d887c4f79a019dbdaeea/RNTester/js/NewAppScreenExample.js#L14

Reply:
Yeah sounds good.